ArduPlane/Plane.cpp

```diff
@@ -884,6 +884,24 @@ bool Plane::set_target_location(const Location &target_loc)
     plane.set_guided_WP(loc);
     return true;
 }
+
+bool Plane::set_target_yaw_rate(const float yaw_rate)
+{
+    if (plane.control_mode != &plane.mode_guided) {
+        // only accept yaw rate updates when in GUIDED mode
+        return false;
+    }
+
+    const auto direction_is_ccw = yaw_rate < 0;
+    if (!is_zero(yaw_rate)) {
+        auto const speed = plane.ahrs.groundspeed();
```

Comment:
Using ground speed here might result in funny shape circles in wind, but maybe that is OK.

Reply:
In-fact, `set_radius_and_direction` does not change the center of the loiter point, so it might do quite odd things.